src/common/AndroidUtils.ts

```diff
@@ -74,78 +72,65 @@ export class AndroidSDKUtils {
     }
 
     public static async androidSDKPrerequisitesCheck(): Promise<string> {
-        return new Promise(async (resolve, reject) => {
-            // Attempt to run sdkmanager and see if it throws any exceptions.
-            // If no errors are encountered then all prerequisites are met.
-            // But if an error is encountered then we'll try to see if it
-            // is due to unsupported Java version or something else.
-            AndroidSDKUtils.fetchAndroidCmdLineToolsLocation([
-                'ignore', // stdin
-                'pipe', // stdout
-                'pipe' // stderr
```

Comment:
Just for my knowledge and want to understand. Why do we not need to pass `ignore, pipe, & pipe` anymore?

Reply:
1. We switched `fetchAndroidCmdLineToolsLocation()` to ultimately call `child_process.exec()` (async), whereas it used to call `child_process.execSync()` (sync), and the async method does not support `StdioOptions`.
2. In this case, `ignore` for `stdin` never really mattered, and `pipe` for `stdout` and `stderr` is effectively the way that `CommonUtils.executeCommandAsync()` operates by default.